<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HSE Sans Black</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>User Rating - Country</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User Rating - Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reviews - Country</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Reviews - Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Price - Country</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Price - Genre</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Year - Country</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Year - Genre</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="bulid_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>HSE Sans Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ПОСТРОИТЬ</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
